### DISCUSSION TITLE
Not able to login Web Dashboard

### DISCUSSION THREAD

**Discussion by prathpatkar** [nextgenhealthcare/connect#4684] [Q&A]
### **Goal**: Access Mirth Administrator console.
1. **Action performed** : I used username and password as 'admin' and 'admin' to login <br>
<img src="https://user-images.githubusercontent.com/75605781/129890487-2db6756e-eccf-4ba7-9336-68a42fe8c1b6.png" width="400"/><br>
2. <b>Action output</b> Invalid login credentials (even the credentials are correct)<br>
3. <b>Action for debugging</b>: view <b>Network tab</b> in Inspect Element.<br>
4. <b>Action result</b>: form is not submitting to server [getting <b>302 found</b> error (i.e. URL is replaced to somewhere)]
<img src="https://user-images.githubusercontent.com/75605781/129891335-9310e3fa-5708-45bf-b68e-0c815061d5d2.png" width="400"/>

Can someone please help me to get out of this situation?

Thanking in anticipation.

**Comment by jonbartels** [comment]
Does the login to the full Administrator client work? If yes look at the `Events` view, are there failed login attempts?

What is the output in `mirth.log` ?

Are you accessing MC directly on a local network by its IP or is it sitting behind something else on the public internet?

**Reply by prathpatkar** [reply]
**QA 1:** I am not able to login from anywhere (nither from **web admin** nor **mirth launcher**).
**QA 2:** last logs are registered at **08/01/2021** but I tried to login today (**08/18/2021**).
 `mirth.log` 

> `ERROR 2021-08-01 04:01:44,942 [Main Server Thread] com.mirth.connect.server.Mirth: http.port port is already in use: 8080
> ERROR 2021-08-01 04:01:44,943 [Main Server Thread] com.mirth.connect.server.Mirth: https.port port is already in use: 8443
> `

**QA 3:** Mirth is present on AWS EC2 and I am accessing it from the local machine by its IP address.

**Reply by jonbartels** [reply]
You either have two instances of MC running *or* there is a genuine port conflict with another application. 

You can change what ports MC listens on by editing `mirth.properties` and the `http.port` and `https.port` options. `8081` and `8444` are common alternatives.
